src/Symfony/Component/Messenger/Bridge/Amqp/Transport/Connection.php

```diff
@@ -258,6 +265,11 @@ private static function normalizeQueueArguments(array $arguments): array
         return $arguments;
     }
 
+    private static function hasCaCertConfigured(array $amqpOptions): bool
+    {
+        return (isset($amqpOptions['cacert']) && !empty($amqpOptions['cacert'])) || 0 !== (int) ini_get('amqp.cacert');
```

Comment:
we try to avoid using empty.

Reply:
Could fabbot detect it ?
(I will fix it anyway, it's just a proposal)